apps/dashboard/app/(app)/semantic-cache/form.tsx

```diff
@@ -28,19 +22,20 @@ const formSchema = z.object({
 });
 
 export default function EnableSemanticCacheForm() {
```

Comment:
please don't use default exports, it can cause issues later when things change

also I don't understand the name here. The implementation looks like it creates a gateway

Reply:
Done